src/types.ts

```diff
@@ -109,9 +109,9 @@ export interface RawPoolEpochRewards {
     starting_block_timestamp: string;
     starting_block_number: string;
     starting_transaction_index: string;
-    ending_block_number?: string;
-    ending_timestamp?: string;
-    ending_transaction_hash?: string;
+    ending_block_number: string;
+    ending_timestamp: string;
+    ending_transaction_hash: string;
```

Comment:
hmmm how? we use this type for other queries too, where these can be undefined / null.

Reply:
rewards only accrue for a finished epoch (known end time)